Makefile

```diff
@@ -33,15 +33,21 @@ clean:
 
 test: clean
 	# Review the CONTRIBUTING documentation for other ways to test.
-	pip install -e . -r requirements/pytorch/devel.txt
-	pip install -r requirements/pytorch/strategies.txt
+	pip install -e . -r requirements/pytorch/base.txt
+	pip install -e . -r requirements/app/base.txt
+	pip install -e . -r requirements/lite/base.txt
```

Comment:
qq: lite doesn't have a `test.txt` requirements file?

Reply:
Yes, there was no need for it for the CI. I'm sure someone will add it once it is needed.